### PR TITLE
Add pull request template

### PR DESCRIPTION
**Pull request by lassoan**
Add a pull request template with instructions for what to do for new extensions and for extension updates. The template file content populates the description field in all new pull requests.

I've tried adding separate templates for extension updates / new extensions but then GitHub did not offer either of them automatically. I had to create a URL manually that contained a template name argument.

### CODE REVIEW

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 1]
```diff
@@ -0,0 +1,28 @@
+# Update an existing extension
+
+You can delete this whole template and just describe which exension is updated and optionally tell us in a sentence what has been changed.
```
Comment (jcfr) [inline .github/PULL_REQUEST_TEMPLATE.md]: exension -> extension

**Comment by fedorov** [comment]
> I've tried adding separate templates for extension updates / new extensions but then GitHub did not offer either of them automatically. I had to create a URL manually that contained a template name argument.

Maybe a simple solution to this could be to have a template that contains 2 sections - one for new extensions, and one for extension updates, and an instruction line that asks the PR contributors to pick the one applicable and remove the other one?

**Comment by lassoan** [comment]
> Maybe a simple solution to this could be to have a template that contains 2 sections

That's exactly what I did

**Comment by jcfr** [comment]
Thanks again @lassoan for adding the template :+1:

**Pull request merged**